pages/components/Header.module.css

```diff
@@ -0,0 +1,12 @@
+.header{
```

Comment:
For now, use tags instead of class names and add CSS to the global sheet, since it is not decided how to move further with styling.

Reply:
Follow basic rules like space between class name and curly brackets.

Reply:
CSS moved to global sheet and used tags instead of class names. Also added space between tags and curly brackets.